src/Diagnostics.DataProviders/DataProviderLogDecorator.cs

```diff
@@ -141,14 +141,14 @@ public Task<string> GetServerFarmWebspaceName(string subscriptionId, string serv
             return MakeDependencyCall(_observerDataProvider.GetServerFarmWebspaceName(subscriptionId, serverFarm));
         }
 
-        public Task<dynamic> GetSite(string siteName)
+        public Task<dynamic> GetSite(string siteName, string slotName = GeoMasterConstants.ProductionSlot)
```

Comment:
One more comment - we should not do this as this will be breaking change. For backward compatibility, you should add another method that takes the slot...

existingMethod(string name) {  Common_Code(name, 'PROD'); }

newMethod(string name, string slot) { {  Common_Code(name, slot); }